R/get-htable-types.R

```diff
@@ -26,4 +26,37 @@ getHtableTypes <- function(data){
   })
   
   as.character(types)
+}
+
+#' Convert to specified class
+#' 
+#' @param x vector
+#' @param cls character
+#' @return converted vector
+#' @seealso https://stackoverflow.com/questions/9214819/supply-a-vector-to-classes-of-dataframe
+toCls = function(x, cls) tryCatch(do.call(paste("as", cls, sep = "."), list(x)),
```

Comment:
If you don't mind, can you make these non-Roxygen docs? Otherwise a spurious entry in the package docs gets created for an internal function that we don't want to export. If you just remove the `'` from `#'` that should do the trick (on this and setHtableClass).